### PR TITLE
Update boilerplate versions

### LINKED ISSUE #19: Update Boilerplate
We wan to update:

- [ ] Dockerfile  and docker packages 
- [ ] Docker compose version

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolve #19 

### Steps to test

1. Run `make run`

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (leisterfrancisco) [inline yarn.lock]: run `yarn` inside of the hapi folder to create the final `yarn.lock` file

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: hapi/package.json [hunk 2]
```diff
@@ -17,31 +17,32 @@
...
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-promise": "^4.3.1",
-    "husky": "4.3.8",
-    "lint-staged": "^10.5.4",
-    "nodemon": "^2.0.4",
-    "prettier": "^2.2.1",
-    "standard": "^16.0.3"
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.0.0",
+    "husky": "^8.0.1",
```
Comment (leisterfrancisco) [inline hapi/package.json:42]: Husky requires a new structure, please, check the [official documentation](https://typicode.github.io/husky/#/?id=install).

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: hapi/package.json [hunk 3]
```diff
@@ -8,40 +8,37 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node src/index.js",
     "watch": "nodemon --enable-source-maps src/index.js",
-    "lint": "lint-staged"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
+    "lint": "lint-staged",
```
Comment (leisterfrancisco) [inline hapi/package.json]: remove this, new husky version handles it different

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Comment by leisterfrancisco** [comment]
@hanakoori01 please update Hasura image to the latest version to support `linux/arm64/v8`, more [here](https://hub.docker.com/layers/graphql-engine/hasura/graphql-engine/v2.11.0-beta.1.cli-migrations-v3/images/sha256-e1ac97c034299853c84928c2abd770758824308a7dcd1f48ce64f31e29527ef6?context=explore)

File: hasura/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM hasura/graphql-engine:v2.0.6.cli-migrations-v3
+FROM hasura/graphql-engine:v2.11.0-beta.1.cli-migrations-v3
```
Comment (xavier506) [inline hasura/Dockerfile:1]: this version change also needs to happen in docker-compose.yml

**Review by xavier506 — CHANGES_REQUESTED** [review]
check hasura version change in docker-compose

**Review by xavier506 — CHANGES_REQUESTED** [review]
lets update docker compose to version 3.8
currently in `docker-compose.yml`
```version: "3.6"```

File: hapi/package.json [hunk 5]
```diff
@@ -8,40 +8,36 @@
...
     "watch": "nodemon --enable-source-maps src/index.js",
-    "lint": "lint-staged"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
+    "prepare": "cd .. && husky install hapi/.husky"
   },
   "license": "MIT",
```
Comment (leisterfrancisco) [inline hapi/package.json:19]: let's add this:

```
"lint-staged": {
    "*.js": [
      "eslint",
      "prettier --write"
    ]
  }
```

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**